packages/airnode-admin/test/e2e/cli.feature.ts

```diff
@@ -66,6 +66,8 @@ describe('CLI', () => {
   beforeAll(() => {
     jest.setTimeout(45_000);
 
+    execSync('yarn build');
```

Comment:
But when you modify the source files and run the e2e tests it will use the outdated build. I think it's better to just build it as it doesn't take too long.

Reply:
I don't agree as this one E2E test would behave differently from the rest. I think expecting a re-build for E2E tests is ok.

Reply:
Reverted.